### DISCUSSION TITLE
[#991927] Entering Editor Mode menu doesn't reset the camera back to First Person

### DISCUSSION THREAD

**Discussion by xKingDark** [Mojang/minecraft-editor#56] [Bugs]
**How to Reproduce**:
  **1.** Switch to Crosshair Mode
  **2.** Change the perspective to Third Person
  **3.** Go back to Editor Mode
  
**Observed Result**:
https://user-images.githubusercontent.com/47607185/224086757-f3ea4740-631e-4069-9fb2-b7fd3c3593a4.mp4

**Expected Result**:
Switches back to First Person

**Helpful Info**:
- Preview build version: `1.19.80.20`

**Comment by minecraftchloe** [comment]
Great find! Updated title with internal ID for tracking.
